lib/ts/recipe/emailpassword/components/features/resetPasswordUsingToken/index.tsx

```diff
@@ -79,7 +79,12 @@ class ResetPasswordUsingToken extends PureComponent<
                       token: this.state.token,
                   };
 
+        const handleBackButtonClick = (): void => {
```

Comment:
Tiny nitpick, but this could be inlined as the other handlers.

Reply:
Updated